common/openapi.yml

```diff
@@ -707,6 +734,22 @@ components:
           type: string
         output_definition:
           type: string
+
+    AbortRunSetResponse:
+      type: object
+      required: [ run_set_id, runs ]
+      properties:
```

Comment:
Maybe we can also return the status of the request? Probably return `Canceling` state? What do you think?

Reply:
I think the `Canceling` state would be goo to return here.